L1Trigger/TrackFindingTMTT/src/Utility.cc

```diff
@@ -0,0 +1,199 @@
+#include "L1Trigger/TrackFindingTMTT/interface/Utility.h"
+#include "L1Trigger/TrackFindingTMTT/interface/TP.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Stub.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Settings.h"
+#include "L1Trigger/TrackFindingTMTT/interface/DeadModuleDB.h"
+
+#include "FWCore/Utilities/interface/Exception.h"
+
+namespace TMTT {
+
+  //=== Count number of tracker layers a given list of stubs are in.
+  //=== By default, consider both PS+2S modules, but optionally consider only the PS ones.
+
+  unsigned int Utility::countLayers(const Settings* settings,
+                                    const vector<const Stub*>& vstubs,
+                                    bool disableReducedLayerID,
+                                    bool onlyPS) {
+    //=== Unpack configuration parameters
+
+    // Note if using reduced layer ID, so tracker layer can be encoded in 3 bits.
+    static bool reduceLayerID = settings->reduceLayerID();
+    // Define layers using layer ID (true) or by bins in radius of 5 cm width (false).
+    static bool useLayerID = settings->useLayerID();
+    // When counting stubs in layers, actually histogram stubs in distance from beam-line with this bin size.
+    static float layerIDfromRadiusBin = settings->layerIDfromRadiusBin();
+    // Inner radius of tracker.
+    static float trackerInnerRadius = settings->trackerInnerRadius();
```

Comment:
Why are these statics? If they continue to be, they should be made `const`.

Reply:
They are now const, no longer static.